[
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/806003314",
    "html_url": "https://github.com/jupyter/notebook/issues/5016#issuecomment-806003314",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/5016",
    "id": 806003314,
    "node_id": "MDEyOklzc3VlQ29tbWVudDgwNjAwMzMxNA==",
    "user": {
      "login": "JerryKurata",
      "id": 13578778,
      "node_id": "MDQ6VXNlcjEzNTc4Nzc4",
      "avatar_url": "https://avatars.githubusercontent.com/u/13578778?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/JerryKurata",
      "html_url": "https://github.com/JerryKurata",
      "followers_url": "https://api.github.com/users/JerryKurata/followers",
      "following_url": "https://api.github.com/users/JerryKurata/following{/other_user}",
      "gists_url": "https://api.github.com/users/JerryKurata/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/JerryKurata/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/JerryKurata/subscriptions",
      "organizations_url": "https://api.github.com/users/JerryKurata/orgs",
      "repos_url": "https://api.github.com/users/JerryKurata/repos",
      "events_url": "https://api.github.com/users/JerryKurata/events{/privacy}",
      "received_events_url": "https://api.github.com/users/JerryKurata/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2021-03-24T17:09:19Z",
    "updated_at": "2021-03-24T17:09:19Z",
    "author_association": "NONE",
    "body": "Same issue on a fresh install and 2 years later.  Any ideas?",
    "performed_via_github_app": null
  }
]
